bin/delete-state-file

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+bucket_name=530003481352-terraform-state
+
+statefile=review-pr-${pr_id}.tfstate
```

Comment:
In bash it's usually more explicit to read from `$1` than to read from a "magic" environment variable